dubbo-serialization/dubbo-serialization-api/pom.xml

```diff
@@ -55,5 +55,9 @@ limitations under the License.
             <groupId>de.ruedigermoeller</groupId>
             <artifactId>fst</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
What is the dependencies of gson? Are all the licenses compatible with Apache?

Reply:
This package is copy from the project dubbo/dubbo-serialization-gson. I think it's OK ?

Reply:
It has not been checked before. When merging into Apache repo, we need to be careful to the dependencies.
Could you list all the dependencies of gson and there license?

Reply:
Hi, I just checked that the gson library has already been added before this pull request. So I think there is no issue here.